include/glow/Backend/BackendUtils.h

```diff
@@ -120,5 +120,41 @@ class RuntimeBundle {
 };
 } // namespace runtime
 
+/// If \p PH is an output placeholder, \returns true.
+/// This is determined by checking if the PH has a user which uses the PH as an
+/// overwritten input.
+bool isOutput(const Placeholder *PH, const Function &F);
+
+/// If \p PH is an input placeholder, \returns true.
+bool isInput(const Placeholder *PH, const Function &F);
+
+/// If \p PH is an output placeholder in the function \p F, \returns true.
+/// This is determined by checking if the PH has a user which uses the PH as an
+/// overwritten input.
+bool isOutput(const Placeholder *PH, const IRFunction &F);
+
+/// If \p PH is an input placeholder in the function \p F, \returns true.
+bool isInput(const Placeholder *PH, const IRFunction &F);
+
+/// Contains information for placeholder during allocation.
+struct PlaceholderInfo {
```

Comment:
The structure is OK. But its name is a bit too general. Let's call it something more specific, e.g. `PlaceholderInputOutputInfo` or something like that.

Reply:
> The structure is OK. But its name is a bit too general. Let's call it something more specific, e.g. `PlaceholderInputOutputInfo` or something like that.

SUre